teams/README.md

```diff
@@ -0,0 +1,80 @@
+
+Review access can be revoked by consensus by the maintainers (except the reviewer in question if they are also a maintainer).
+
+### Committers
+
+Committers are individuals who are granted the write access to the repositories belong to the team. A committer is usually responsible for a certain area or several areas of the code where they oversee the code review process. The area of contribution can take all forms, including code contributions and code reviews, documents, education, and outreach. Committers are essential for a high quality and healthy project.
+
+Commit access is by invitation only and must be approved by consensus of the maintainers. A committer is considered emeritus by their own declaration. An emeritus committer may request reinstatement of commit access from the maintainers, which will be sufficient to restore him or her to active committer status.
+
+Commit access can be revoked by consensus by the maintainers (except the committer in question if they are also a maintainer).
+
+### Maintainers
+
+The maintainers consist group of active committers that moderate the discussion, manage the project release, and proposes new committers or maintainers. Potential candidates are usually proposed via an internal discussion among maintainers, followed by a consensus approval, i.e. a concrete number of approvals, and no vetoes. Any veto must be accompanied by reasoning. Maintainers should serve the community by upholding the community practices and guidelines TiDB a better community for everyone. Maintainers should nominate new reviewer, committers and maintainers, and should also strive to only nominate new candidates outside of their own organization.
+
+Membership of the maintainers is by invitation only and must be approved by a consensus of the maintainers. A maintainer is considered emeritus by their own declaration. An emeritus member may request reinstatement to the maintainers, which will be sufficient to restore him or her to active maintainers.
+
+Membership of the maintainers can be revoked by a consensus vote of all the maintainers other than the member in question.
+
+## Decision Making
+
+Within the teams, different types of decisions require different forms of approval. For example, the previous section describes several decisions which require 'consensus' approval. This section defines how voting is performed, the types of approvals, and which types of decision require which type of approval.
+
+### Voting
+
+Decisions regarding the project are made by votes on [the community repository](https://github.com/pingcap/community). Votes are clearly indicated by a pull request adding an entry under [votes](/votes/README.md) folder. Votes may contain multiple items for approval and these should be clearly separated. Voting is carried out by replying to the vote pull request. Voting may take three flavors
```

Comment:
I'm going to join community level decision making folder, team level decision making folder, and legacy rfcs as a follow up.

Reply:
RFCs are also votes, toc proposals are also votes. We don't have to make many folders separately.